src/lib/parser.ts

```diff
@@ -1,29 +1,75 @@
 import { Face } from '../types/Face'
 import { Vertex } from '../types/Vertex'
 
-export function parseVertices(input: string): Vertex[] {
+type ParsedLine = string[]
```

Comment:
what does this line mean?

Reply:
it's a little simplification so we don't have to use string[][] typing down the road

Reply:
`string[][]` represents all lines in the file, where each inner `string[]` contains the fields for a single line (id, x, y, z)